### PR TITLE
Updating Readme for the project

### PR DESCRIPTION
**Pull request by palashjain95**
1. Updating Readme for the project
2. Returning nil incase the tracing headers are not initialised

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
-# wavefront-go-middleware
\ No newline at end of file
+# wavefront-go-middleware [![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-go-middleware.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-go-middleware) [![Go Report Card](https://goreportcard.com/badge/github.com/wavefrontHQ/wavefront-go-middleware)](https://goreportcard.com/report/github.com/wavefrontHQ/wavefront-go-middleware) [![OpenTracing Badge](https://img.shields.io/badge/OpenTracing-enabled-blue.svg)](http://opentracing.io)
+
+This library provides wavefront-opentracing-middleware support for various Go's web framework. The middleware takes care of the entire lifecycle of span reporting with minimal code injection in Api's. It provides a custom logger abstracted on top of Go standard Logger which injects request-scoped unique trace identifiers into logs generated while servicing an Api request.
```
Comment (vikramraman) [inline README.md]: Some minor nits: 
* for various Go web frameworks.
* The middleware handles the entire lifecycle of reporting spans with minimal code injection within APIs.

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+# wavefront-go-middleware [![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-go-middleware.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-go-middleware) [![Go Report Card](https://goreportcard.com/badge/github.com/wavefrontHQ/wavefront-go-middleware)](https://goreportcard.com/report/github.com/wavefrontHQ/wavefront-go-middleware) [![OpenTracing Badge](https://img.shields.io/badge/OpenTracing-enabled-blue.svg)](http://opentracing.io)
+
+This library provides wavefront-opentracing-middleware support for various Go's web framework. The middleware takes care of the entire lifecycle of span reporting with minimal code injection in Api's. It provides a custom logger abstracted on top of Go standard Logger which injects request-scoped unique trace identifiers into logs generated while servicing an Api request. 
+
+## Requirements
+-   Go 1.10 or higher.
+
+## Usage
+
+- <a href="https://echo.labstack.com"><img height="20" src="https://cdn.labstack.com/images/echo-logo.svg"></a>
```
Comment (vikramraman) [inline README.md]: Make this a sub header than a bullet point:
```
### <a href="https://echo.labstack.com"><img height="20" src="https://cdn.labstack.com/images/echo-logo.svg"></a>
```

File: README.md (hunk 1 above)
Comment (palashjain95) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (palashjain95) [reply README.md]: Done

**Review by vikramraman — APPROVED** [review]

**Pull request merged**